understanding/identify-input-purpose.html

```diff
@@ -110,9 +110,9 @@ <h2>要約</h2>
             </section>
             <section id="intent">
                <h2>意図</h2>
-               <p>この達成基準の意図は、利用者に関する情報を収集するフォームの入力の目的を、プログラムによる解釈を可能にして、その結果、ユーザエージェントが異なるモダリティを使用し、その目的を抽出して利用者に提示できるようにすることである。特定のフィールドで期待される特定の種類のデータをプログラムで宣言する機能により、とりわけ特定の認知障害を持つ人々にとって、フォームへの入力が容易になる。
+               <p>この達成基準の意図は、利用者に関する情報を収集するフォームの入力目的がプログラムによる解釈を可能にすることであり、その結果、ユーザエージェントが様々なモダリティを使用する利用者に対して、この目的を抽出して提示できるようにすることである。特定のフィールドで期待される特定の種類のデータをプログラムで宣言する機能により、とりわけ特定の認知障害を持つ人々にとって、フォームへの入力が容易になる。
                </p>
-               <p>適切なラベルと説明は、利用者が入力フォームフィールドの目的を理解するのに役立つが、利用者は、特定の種類の情報を収集するフィールドを、さまざまなモダリティに合わせて曖昧さがなく一貫性がありカスタマイズ可能な方法で (ユーザエージェントのデフォルトで、又は支援技術の助けを借りてのどちらかで) レンダリングできるという利点がある。
+               <p>適切な可視のラベル及び説明は、利用者がフォームの入力フィールドの目的を理解するのに役立つが、特定の種類の情報を収集するフィールドが、様々なモダリティ (ユーザエージェントのデフォルトの設定、又は支援技術による支援のいずれかを通じて) に合わせて、明確で、一貫性があり、そして場合によってはカスタマイズされた方法でレンダリングされることによって、利用者は恩恵を受ける場合がある。
```

Comment:
これ原文が難解すぎてヤバイですね。

要するに入力目的がマシンリーダブルだとブラウザや支援技術がいい感じにカスタマイズ表示でき、それでユーザーが恩恵を受けることがあるという話ですが……

an unambiguous, consistent, and possibly customized way for different modalities の部分ですが、different modalities は 全体ではなく最後の箇所だけにかかっているものと思います。つまり「明確で、一貫性があり、そして場合によっては様々なモダリティに合わせてカスタマイズされた方法でレンダリング」かと。

そしてカッコ書きになっている部分は「レンダリングされる」にかかっているものと思います。このカッコ書き、原文だとハイフン区切りで末尾に置いてあるので、同様の処理にしたいですね…… (カッコ書きにして中に入れると、ただでさえ長くて読みにくいものがさらに読みにくくなるので)。

また having fields be rendered なので「フィールドをレンダリングされた状態にさせる」というような使役のニュアンスがありそうですね。「レンダリングさせる」?

あまりにもアレなので2文に切ることを検討してもいいかもしれませんが、1文のままでやるとすると、たとえばこうでしょうか?

```suggestion
               <p>適切な可視のラベル及び説明は、利用者がフォームの入力フィールドの目的を理解するのに役立つが、特定の種類の情報を収集するフィールドを、明確で、一貫性があり、そして場合によっては様々なモダリティに合わせてカスタマイズされた方法でレンダリングさせることは、利用者に恩恵をもたらす場合がある -  これはユーザエージェントのデフォルトの設定、又は支援技術の助けによって実現される。
```